src/components/inline-search-form/inline-search-form.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from "react";
+
+import cn from "classnames";
+
+import { Button, Input } from "@mtfh/common";
+
+interface Props {
+  onSubmit: (searchText: string) => void;
+  className?: string;
+  loading: boolean;
+}
+
+// Cant use form tags inside existing form
+export const InlineSearchForm = ({ onSubmit, className, loading }: Props) => {
+  const [input, setInput] = useState<string>("");
+  const [error, setError] = useState<string | null>(null);
+
+  const handleSubmit = () => {
+    if (loading) return;
+
+    if (input === "" || input.length < 2) {
+      setError("You must enter at least 2 characters.");
+      return;
+    }
+
+    setError(null);
+    onSubmit(input);
+  };
+
+  return (
+    <div className={className || ""}>
+      <div
+        className={cn("govuk-form-group  lbh-form-group visually-hidden-label", {
+          "govuk-form-group--error": error,
+        })}
+      >
+        {error && (
+          <span className="govuk-error-message lbh-error-message" aria-hidden="true">
+            {error}
+          </span>
+        )}
+
+        <Input
+          style={{ maxWidth: "100%" }}
+          onKeyPress={(e) => {
+            if (e.key === "Enter") {
+              e.preventDefault();
+              handleSubmit();
+            }
+          }}
```

Comment:
Can this logic be moved into handleSubmit?

Reply:
Created a handleKeyPress method which then calls handleSubmit